conda/recipes/kvikio/meta.yaml

```diff
@@ -28,6 +28,8 @@ requirements:
     - setuptools
     - cython >=0.29,<0.30
     - cudatoolkit {{ cuda_version }}.*
+    - cmake >=3.20.1
```

Comment:
Should `cmake` be in `build`? Typically that is where build tools (like `cmake`) are placed. Though please let me know if there is a reason scikit-build prefers `cmake` in `host`

Reply:
Hmm that is a good question. I think `build` makes the most sense, but I put it in `host` because that's where all the build tools were previously placed (`setuptools`, `cython`). Should I move all of them over?

Reply:
Looking at the current list, I think _technically_ we would want
```
  build:
    - setuptools
    - cython >=0.29,<0.30
    - cmake >=3.20.1
    - scikit-build >=0.13.1
    - ninja
  host:
    - cudatoolkit {{ cuda_version }}.*
    - python
    - libkvikio {{ version }}
```
But in practice we create separate packages for every platform so everything can safely go in `build`. Does that sound right to you? Also is `python` necessary at all? I don't see `Python.h` anywhere, so I assume we don't actually need to link to the Python libraries (except as handled by Cython), but I guess no harm in including it.

Reply:
Only command line tools (like `cmake` and `ninja`) would move. Things that are linked against or imported (like `python`, `cython`, and `setuptools`) would stay in `host`.